gradle.properties

```diff
@@ -9,7 +9,7 @@ modBaseName=minecolonies
 javaVersion=17
 useJavaToolChains=true
 fml_range=[45,)
-forge_range=[46.0.1,47.1.3]
+forge_range=[46.0.1]
```

Comment:
Is it intended that minecolonies will only accept Minecraft 1.20 going forward? Or does NeoForge use 46.x.x for 1.20.1?